learning-development/accessibility.qmd

```diff
@@ -52,6 +51,48 @@ For more advice about colour in charts and visualisations specifically, includin
+
+In addition to accessibility, the guidance also addresses usability and machine readability. While these areas often overlap with accessibility, there are instances where they may conflict. Depending on user needs, it might be necessary to produce separate versions of a spreadsheet—one for human use and another optimized for machine readability—to ensure both accessibility and functionality are maintained.
+
+The [guidance](https://analysisfunction.civilservice.gov.uk/policy-store/releasing-statistics-in-spreadsheets/) is extensive, including sections how to format sheets and cells, accessible titles, using formula & macros and much more!
+
+## b. Automate Accessibility During the Coding Process
+
+Automation is a powerful way to ensure that accessibility is consistently applied throughout the spreadsheet creation process, especially when working with large datasets or producing numerous spreadsheets. By integrating accessibility into the coding process, you can streamline your workflow and reduce the chances of missing critical accessibility features.
+
+Here’s how you can automate accessibility using specific tools in Python and R:
+
+1. For Python Users – gptables Package:
+
+- Overview: The ``gptables`` package is designed to help Python users create accessible tables within spreadsheets. This package automates the formatting of tables to meet accessibility standards, ensuring that key elements such as table headers, data descriptions, and structures are correctly implemented.
+- Benefits: By using ``gptables``, you can automatically generate tables that are readable by screen readers, properly formatted for navigation, and compliant with accessibility regulations. This reduces the manual effort required to adjust table formats and enhances consistency across multiple spreadsheets.
+
+2. For R Users – ``a11ytables`` Package:
+
+- Overview: The ``a11ytables`` package in R serves a similar purpose by helping users create accessible tables within their spreadsheets. The package includes functions that facilitate the creation of tables with correct headers, descriptive text, and appropriate structures, making them accessible to all users, including those relying on assistive technologies.
+- Benefits: Using ``a11ytables`` ensures that your spreadsheets are automatically optimized for accessibility, minimizing the need for manual intervention. It’s particularly useful when producing multiple tables, as it maintains consistent formatting and accessibility standards throughout your documents.
+
+Why Automate?
```

Comment:
Again, I'd set this as a 4th tier header:
#### Why automate?

And I'd move it earlier, to around the current line 73.